src/parser/csql_grammar.y

```diff
@@ -5341,31 +5395,53 @@ meta_class_spec
 			  {
 			    ocs->info.spec.entity_name = $2;
 			    ocs->info.spec.only_all = PT_ONLY;
-			    ocs->info.spec.meta_class = PT_CLASS;
+			    ocs->info.spec.meta_class = PT_META_CLASS;
 			  }
 
-			if (ocs)
-			  ocs->info.spec.meta_class = PT_META_CLASS;
 			$$ = ocs;
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
 		DBG_PRINT}}
 	;
 
-only_all_class_spec
-	: only_class_name opt_partition_spec
-		{{ DBG_TRACE_GRAMMAR(only_all_class_spec, : only_class_name opt_partition_spec );
-
-			PT_NODE *ocs = parser_new_node (this_parser, PT_SPEC);
+only_all_class_spec_with_server
+	: class_name_with_server_name opt_partition_spec
```

Comment:
I think the opt_partition_spec rule should be excluded.

Reply:
It's fixed.